framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -330,6 +333,106 @@ public void processClassTree(ClassTree classTree) {
         super.visitClass(classTree, null);
     }
 
+    /**
+     * Check that the field invariant meets the following requirements:
```

Comment:
"refined field invariant" is confusing to me.  What's being refined, the field or the invariant? It's especially confusing since it's not mentioned anywhere else in describing "field invariants".  Though maybe when you review my documentation improvements you'll suggest some other places that it might go.